src/components/vr_cooperation/src/vr_module.cc

```diff
@@ -308,5 +316,18 @@ void VRModule::UnregisterRequest(uint32_t correlation_id) {
   request_controller_.DeleteRequest(correlation_id);
 }
 
+void VRModule::RegisterService(application_manager::MessagePtr msg) {
```

Comment:
This method should creaqe notification command and send response. All logic for response should be in response message.